src/styles/utils/_vars.scss

```diff
@@ -1 +1,7 @@
-$c-gray: #eee;
+$metMainColor: #d12d35;
```

Comment:
Consider using more descriptive variable names for your color variables. For example, use 'primaryColor' instead of 'metMainColor', 'primaryColorHover' instead of 'metMainColorHover', and so on.